src/pages/functions/getEvents.ts

```diff
@@ -0,0 +1,15 @@
+export default function getEvents(orgId : string | string[]) {
```

Comment:
As far as I understand it means string or string in an array. I got the following error when only using `string` as a prop type:

error TS2345: Argument of type 'string | string[]' is not assignable to parameter of type 'string'.

Without knowing how, I assumed it has to do with the dynamic routing in next.

Reply:
That's interesting! Sounds like a good hypothesis, but could probably do with some more research.

The problem I see with this function as it's currently implemented is that the type annotation declares that it can take a `string` or an array of `string`. But the implementation of the function does not actually handle the case where an array of strings is passed to it.

The purpose of type annotations is to tell the programming language / compiler that as long as a certain type is used, it's reasonable not to expect any bugs to occur. Basically this is saying to typescript that "as long as `orgId` is either a `string` or a `string[]`, all should be fine!".

But then on line 3, the variable `orgId` is injected into a string. If `orgId` is an array of strings, what will be the result on line 3?

Reply:
> But then on line 3, the variable `orgId` is injected into a string. If `orgId` is an array of strings, what will be the result on line 3?

The `id` of an organization is always a `number`, so when being matched with an `array` of `string` instead of a `string` number from the url path via `context.params`, the result will be a 404.